src/link-manager.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @fileoverview Link manager for controlling the link query.
+ */
+
+/** @const */
+genotet.linkManager = {};
+
+/**
+ * Object that stores the link views by their type.
+ */
+/** @type {!Object<string, !Array<genotet.View>>} */
```

Comment:
If the key of an object is of type _string_, then you can omit it.
Object<string, T> is the same as Object<T>